spec/requests/claims_spec.rb

```diff
@@ -1,6 +1,14 @@
 require "rails_helper"
 
 RSpec.describe "Claims", type: :request do
+  describe "claims#new request" do
+    it "renders the correct template with `current_claim` set correctly" do
+      claim = Claim.new
+      expect(Claim).to receive(:new) { claim }
```

Comment:
Can we test this new action without having to mock out the call to `Claim.new`? It makes the spec test an internal implementation detail, rather than the outcome that we expect. Is the underlying outcome that we want that the form on the page will `POST` to the `create` action?